src/twisted/conch/manhole.py

```diff
@@ -151,8 +153,9 @@ def _ebDisplayDeferred(self, failure, k, obj):
         return failure
 
 
-    def write(self, data, async=False):
-        self.handler.addOutput(data, async)
+    def write(self, data, _async=None, **kwargs):
```

Comment:
instead of _async, can this be renamed to `asAsync=True|False` ? ...

Reply:
+1, I like ``asAsync`` better than ``_async``. I've seen ``async_`` used in other libraries and it matches the PEP-8, but this name does not pass Twisted style check.